rpm/fedora/xournalpp.spec

```diff
@@ -34,6 +34,7 @@ BuildRequires:  pkgconfig(lua)
 BuildRequires:  pkgconfig(poppler-glib)
 BuildRequires:  pkgconfig(portaudiocpp) >= 12
 BuildRequires:  pkgconfig(sndfile)
+BuildRequires:  pkgconfig(gtksourceview) >= 3.18 
```

Comment:
This might also be wrong -- it might need to be, instead `BuildRequires:  pkgconfig(gtksourceview4)`...

Reply:
I can try to test the modified `rpm` over the weekend.

Reply:
Changed to `pkgconfig(gtksourceview-4)`